gatorgrade/main.py

```diff
@@ -16,6 +19,8 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
+        checks = parse_config(FILE)
+        checks = run_and_display_command_checks(FILE)
```

Comment:
If you take a look at PR #68, you will see that the `run_and_display_command_checks` function does not take a file path as its argument; instead, it takes the list of checks that should be run and these checks are the output of the `parse_config` function. So, this code needs to be changed such that the argument passed to `run_and_display_command_checks` is the output of `parse_config`.